optimization/tests/GridCreator.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include <vector>
 
+
```

Comment:
I suppose this was unintended?

Reply:
Sort of. I like to have an extra space between headers and code, but sure, let's mark it as unintentional :D It shouldn't matter with the formatter, but it does indeed add one extra line the reviewer has to look at... :)

Reply:
Alright nvm then. As long as the stylechecker doesn't complain I'm fine with it :)